vignettes/r-and-js.Rmd

```diff
@@ -99,12 +99,16 @@ unlink(tp, recursive = TRUE)
 # df = data.frame(lw=runif(length(sfc), min=1, max=5))
 run_tgve = function(layerName="line") {
   setup(p)
-  gj = geojsonsf::sf_geojson(st_as_sf(sfc), simplify = FALSE)
-  ip = build(tp, data = gj,
+  sf = st_as_sf(sfc)
+  ps = explore_sf(sf, background = TRUE)
```

Comment:
this is what is possible @rolyp when we avoid pasting json into an app source for browsers to read.